docs/references/api/deviceshifu-configmap.md

```diff
@@ -79,6 +79,27 @@ instructions: |
       ... # You can continue to configure commands and corresponding NodeID according to your own needs, just continue to add according to this format
 ```
 
+## DeviceShifugatewayPropertyList
+
+Shifu communicates with the LeShan server through the LwM2M protocol, demonstrating seamless device management and interaction, which is particularly important in edge computing environments with multi-protocol and multi-type IoT devices. The following shows how to achieve other protocol device access through a gateway. For more examples, please refer to the [examples](https://github.com/Edgenesis/shifu/tree/main/examples).
```

Comment:
The Shifu gateway not only works with the LwM2M protocol, but also converts the device hifu northbound protocol to other protocols.